angular/projects/spark-angular/src/lib/components/sprk-modal/sprk-modal.module.ts

```diff
@@ -2,11 +2,30 @@ import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
 import { SprkIconModule } from '../sprk-icon/sprk-icon.module';
+import { SprkStackItemModule } from '../../directives/sprk-stack-item/sprk-stack-item.module';
+import { SprkButtonModule } from '../../directives/sprk-button/sprk-button.module';
```

Comment:
wow you even updated the module file and did the exports! haha this is the best!